server/src/test/java/org/spine3/server/aggregate/AggregateRepositoryShould.java

```diff
@@ -399,7 +409,8 @@ protected TestAggregateRepository(BoundedContext boundedContext) {
         /** Needs to be static. */
         private static final Map<CommandId, Command> commandsHandled = newHashMap();
 
-        @SuppressWarnings("PublicConstructorInNonPublicClass") /** It is required to be public. */
+        @SuppressWarnings("PublicConstructorInNonPublicClass")
+        /** It is required to be public. */
```

Comment:
This shouldn't probably be a Javadoc comment, but a comment for warning suppression, which comes on the same line. The excessive star was an error in the first place.